src/main/java/edu/hawaii/its/groupings/configuration/OotbCustomUserDetails.java

```diff
@@ -0,0 +1,24 @@
+package edu.hawaii.its.groupings.configuration;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.security.core.GrantedAuthority;
+
+import edu.hawaii.its.groupings.access.UhAttributes;
+import edu.hawaii.its.groupings.access.User;
+
+public class OotbCustomUserDetails extends User {
+    public OotbCustomUserDetails(String username, String password, Collection<GrantedAuthority> authorities,
+            String givenName) {
+        super(username, password, authorities);
+        Map<String, Object> defaultAttributes = new HashMap<>();
+        defaultAttributes.put("cn", Arrays.asList("James"));
+        defaultAttributes.put("mail", Arrays.asList("example@example.com"));
```

Comment:
These two attrributes should be set in the properties file too. I'm not sure if the "mail" is being used by the Feedback page. Would be interesting to know.